documentation/assemblies/overview/assembly-security-overview.adoc

```diff
@@ -10,6 +10,7 @@ A secure deployment of Strimzi can encompass:
 * Encryption for data exchange
 * Authentication to prove identity
 * Authorization to allow or decline actions executed by users
+* Compliance with FIPS (Federal Information Processing Standards) to ensure data security and system interoperability.
```

Comment:
I would probably avoid the _compliance_ word and use something like _Support for FIPS-enabled Kubernetes cluster_.